.changeset/common-candies-kiss.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/create-app': minor
```

Comment:
`patch` for this one, we only consider the CLI command itself when it comes to stability of `create-app`

Reply:
Ok this is now a patch.